### ISSUE TITLE
logo url not shown on metadata pages

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/checklistbank#1105]
maybe the property has changed some time back, but Dataset.logo does not show here:
https://www.checklistbank.org/dataset/30477/about
although it exists: https://api.checklistbank.org/dataset/30477.json

**Comment by thomasstjerne** [comment]
We used to be able to CRUD logos at `dataset/${datasetKey}/logo` how do we do this after the change of the dataet/image scope?

**Comment by mdoering** [comment]
It used to be `/image/{key}/logo`, now it is `/dataset/{key}/logo` with support for POST, GET & DELETE. I see PUT is missing, but that never existed before either. I could quickly add that if it's useful.

**Comment by mdoering** [comment]
and then there is `/dataset/{key}/logo/source/{id}` but GET only

**Comment by thomasstjerne** [comment]
I dont think we need PUT - I´ll test a bit more, couldn´t get `/dataset/{key}/logo` to work before

**Comment by thomasstjerne** [comment]
Seems neither of them work 
https://api.checklistbank.org/dataset/30477/logo
https://api.checklistbank.org/image/30477/logo

**Comment by mdoering** [comment]
you can see the logo files also here: https://download.checklistbank.org/images/dataset/
There is no 30477 logo.

Ah, the original file is a SVG, maybe that causes problems for the inserter:
https://unite.ut.ee/img/unite-logo-web.svg

Looking at the original issue, the problem is not that the logo _image_ is not showing, but that the logo URL is not showing below in the metadata under `Logo Url`

**Comment by mdoering** [comment]
added svg support to the backend
